tidb-cloud/data-service-manage-endpoint.md

```diff
@@ -62,10 +109,10 @@ On the right pane of the endpoint details page, you can click the **Properties**
     - Minimum value: `1`
     - Unit: millisecond
 
-- **Max Rows**: the maximum number of rows that the endpoint returns.
+- **Max Rows**: the maximum number of rows that the endpoint can operate or return.
 
     - Default value: `50`
-    - Maximum value: `2000`
+    - Maximum value: `2000` for batch-disabled endpoints and `100` for batch-enabled endpoints
```

Comment:
2000 when batch operations are disabled and 100 when batch operations are enabled for the endpoints